scheds/rust/scx_layered/src/bpf/intf.h

```diff
@@ -36,7 +36,7 @@ enum consts {
 	USAGE_HALF_LIFE		= 100000000,	/* 100ms */
 
 	HI_FALLBACK_DSQ		= MAX_LAYERS * MAX_DOMS,
```

Comment:
Maybe rename to sth like `HI_FALLBACK_DSQ_BASE` to indicate that it's not just single DSQ?